ManatiCR/retopais
# RP-44: Sponsors categories.

https://manati.atlassian.net/browse/RP-44

How-to-test:
- Create 3 alliances (type: sponsor level 1, sponsor level 2, sponsor level 3)
- Go to homepage, you should get those sponsors with different image styles

Other minor changes:
- Labels for logo in alliance node form
- Fix function name that was causing a php fatal error
- Fix package name and add project path for site layout feature
- Added sponsor1, sponsor2 and sponsor3 image styles (they need to be edited during theming phase)
